include/roboteam_ai/stp/evaluations/global/BallCloseToThemGlobalEvaluation.h

```diff
@@ -2,18 +2,18 @@
 // Created by jordi on 06-05-20.
 //
 
-#ifndef RTT_BALLCLOSETOTHEMINVARIANT_H
-#define RTT_BALLCLOSETOTHEMINVARIANT_H
+#ifndef RTT_BALLCLOSETOTHEMGLOBALEVALUATION_H
+#define RTT_BALLCLOSETOTHEMGLOBALEVALUATION_H
 
 #include <NFParam/Param.h>
```

Comment:
wut do we still use this?

Reply:
It came through with the refactoring. Shall i remove all occurrence of such ifndef? Assuming you mean that with your comment.

Reply:
I think he means specifically this invariant/evaluation, and im guessing we might (want to) use it somewhere?